taskcat/stacker.py

```diff
@@ -1755,6 +1755,8 @@ def aws_api_init(self, args):
             either profile name, access key and secret key or none.
         """
         print('\n')
+
+        self.set_default_region = ClientFactory().get_default_region(args.aws_access_key, args.aws_secret_key, None, args.boto_profile)
```

Comment:
you'll be pleased to see I'm reusing this everywhere, not just 2/3

Reply:
I don't understand the use case for overriding the function. Did you intead this to read either of these? 
-  `self.region = ClientFactory().get_default_region(args.aws_access_key, args.aws_secret_key, None, args.boto_profile)`
- `self.set_default_region(region=ClientFactory().get_default_region(args.aws_access_key, args.aws_secret_key, None, args.boto_profile)` here?